src/main/java/com/arpnetworking/clusteraggregator/Main.java

```diff
@@ -98,20 +123,10 @@ public static void main(final String[] args) {
                 .addTrigger(new FileTrigger.Builder().setFile(configurationFile).build())
                 .addListener(configurator)
                 .build();
+        configRef.set(configuration);
```

Comment:
I'll compromise.  I'll set them immediately after construction, but I won't do away with the local variables for them.